webapp/src/containers/SwapPage/service.tsx

```diff
@@ -37,7 +37,7 @@ export const handleTestPoolSwapTo = async (formState) => {
     const testPoolSwapAmount = await rpcClient.testPoolSwap(
       address1,
       formState.hash1,
-      Number(formState.amount1),
+      new BigNumber(formState.amount1).toNumber(),
```

Comment:
Don't convert to BigNumber and then back to Number, we're not achieving anything by doing this